assets/js/blocks/cart-checkout/cart/full-cart/checkout-button/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import Button from '@woocommerce/base-components/button';
+import { CHECKOUT_URL } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import { AmericanExpressLogo } from './american-express-logo';
+import './style.scss';
+import { MastercardLogo, VisaLogo } from './payment-logos'; // @todo we want to import this from `@automattic/composite-checkout` when it's published in NPM
+
+/**
+ * Checkout button rendered in the full cart page.
+ */
+const CheckoutButton = () => {
+	return (
+		<Fragment>
+			<Button
+				className="wc-block-cart__submit-button"
+				href={ CHECKOUT_URL }
+			>
+				{ __( 'Proceed to Checkout', 'woo-gutenberg-products-block' ) }
+			</Button>
+			<div className="wc-block-cart__payment-methods">
+				<MastercardLogo />
+				<AmericanExpressLogo />
+				<VisaLogo />
+			</div>
+		</Fragment>
+	);
+};
+
+export default CheckoutButton;
```

Comment:
Just to make sure we are on the same page: you mean using the components from Gutenberg or creating our own?

I didn't know about Gutenberg's `<Card>` and I think it would be good to use it here, but it looks like it's not released yet in the last version of `@woocommerce/components` so we should postpone this for a follow-up until it's released?

Reply:
> you mean using the components from Gutenberg or creating our own?

Ideally from `@wordpress/components`, it's available via directly consuming the package, but you're right we can't use it for the admin context (via `wp.components` external) because we're reliant on the WP version.

I think it'd be good to start using the patterns that will be available to us even if that means we temporarily drop a copy of the component in our package until we can reliably consume it directly from its source?

I have an idea.  I wonder if we alias `@wordpress/components` to a different package name, and use that different package name in bundles, it would allow us to import directly from the package for anything we want to use regardless of WP version (so basically what's available from the component package itself, which we can keep up to date independently)?  If you agree, I'm okay with this being experimented on outside this pull.  But regardless of whether it works or not, I'd like to see us try and implement the pattern of a `<Card />` panel that will help keep things consistent in our layouts.  Thoughts?

Reply:
Completely agree. :+1:  I created an issue: #1370.